packages/contracts/src/resources/site.ts

```diff
@@ -207,17 +236,21 @@ type ProcessedPubBase = {
 };
 
 export type ProcessedPub<Options extends MaybePubOptions = {}> = ProcessedPubBase & {
+	/**
+	 * Is an empty array if `withValues` is false
+	 */
```

Comment:
i chose not to do the annoying thing i was doing before and make the values `?: never` when `withValues: false` is set, because too many places rely on there being some sort of values